packages/bruno-cli/src/commands/run.js

```diff
@@ -200,6 +211,10 @@ const builder = async (yargs) => {
       type: 'boolean',
       description: 'Allow insecure server connections'
     })
+    .option('tests-only', {
+      type: 'boolean',
+      description: 'Only run requests that have a test'
```

Comment:
If it runs also in case there are assertions only I would extend this description to be clear when request is executed.